Swift/Helpers/FSExtensions/FSE+String.swift

```diff
@@ -71,6 +71,29 @@ public extension String {
         return ceil(rect.size.height)
     }
     
+    public func fs_stringBetweenString (firstString: String, secondString: String) -> String? {
+        var string: String?
+        var range: Range<Index>? = nil
+        
+        range = self.rangeOfString(firstString)
+        if let lRange = range {
+            let strinBefore = self.substringToIndex(lRange.endIndex)
+            string = self.stringByReplacingOccurrencesOfString(strinBefore, withString: "")
+        } else {
+            return nil
+        }
+        
+        range = string?.rangeOfString(secondString)
+        if let lRange = range {
+            let stringAfter = string?.substringFromIndex(lRange.startIndex)
+            string = string?.stringByReplacingOccurrencesOfString(stringAfter!, withString: "")
+        } else {
+            return nil
+        }
+        
+        return string
```

Comment:
Maybe will be better to get firstString endIndex and secondString startIndex and just get substring in range?